restapi/src/main/java/io/stargate/web/docsapi/resources/DocumentResourceV2.java

```diff
@@ -63,19 +62,9 @@
   @Inject private Db dbFactory;
   private static final Logger logger = LoggerFactory.getLogger(DocumentResourceV2.class);
   private static final ObjectMapper mapper = new ObjectMapper();
```

Comment:
this can also be injected as well right? It's in the env..

Reply:
Yes - I just missed this somehow! Thanks